apps/electron-app/src/renderer/src/components/layout/NavigationBar.tsx

```
@@ -161,166 +190,156 @@ const NavigationBar: React.FC = () => {
         }
 
         // Get real browsing history from saved contexts
-        const contexts = await window.vibe.content.getSavedContexts();
+        const contexts =
+          (await window.vibe?.content?.getSavedContexts?.()) || [];
+        console.log("Found contexts:", contexts.length);
         const historyMatches = contexts
           .filter(
-            ctx =>
+            (ctx: any) =>
               (ctx.url && ctx.url.toLowerCase().includes(inputLower)) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)